plugins/modules/device_credential_workflow_manager.py

```diff
@@ -2003,7 +2094,12 @@ def get_want_assign_credentials(self, AssignCredentials):
                         self.msg = "The username and description of the CLI credential are invalid"
                         self.status = "failed"
                         return self
-                want.get("assign_credentials").update({"cliId": cliDetail.get("id")})
+                if self.get_ccc_version_as_integer() <= self.get_ccc_version_as_int_from_str("2.3.5.3"):
```

Comment:
if it is used in multiple places, then 
current_ccc_version_as_int = self.get_ccc_version_as_integer()

Reply:
added current_ccc_version_as_int = self.get_ccc_version_as_integer() in def get_want_assign_credentials(self, AssignCredentials) where it is used multiple times